src/components/Transactions/BlockExplorer.tsx

```diff
@@ -0,0 +1,37 @@
+import { ExportOutlined } from '@ant-design/icons';
+
+import { shortAddress } from '../../utils';
+import { ChainId, NetworkScanUrl, BlockchainExplorer } from '../../constants';
+interface IProps {
+  address: string;
+  chars?: number;
+  chainId: number;
+  isCheckingTx: boolean;
```

Comment:
Remove this (why did you call it `checking`? are we checking anything here?). Instead, you should replace this and `address` with something like:
```
hash: string;
type: 'address' | 'tx';
```